scripts/test_docker_compose.sh

```diff
@@ -5,7 +5,7 @@ cd test_docker
 sleep 2
 ./iroha_client_cli account register --name="Alice" --domain="Soramitsu" --key="ed0120a753146e75b910ae5e2994dc8adea9e7d87e5d53024cfa310ce992f17106f92c"
 sleep 2
-./iroha_client_cli asset register --name="XOR" --domain="Soramitsu"
+./iroha_client_cli asset register --name="XOR" --domain="Soramitsu" --value_type Quantity
```

Comment:
It is better to follow the same style in one CLI invocation, so better to convert to `--value_type="Quantity"`